### ISSUE TITLE
import-destructuring-spacing not found

### ISSUE THREAD

**Issue by lunanigra** [palantir/tslint#2486]
### Bug Report

- __'import-destructuring-spacing' not working after update__
- __tslint 5.0.0__
- __Has been working before update. Other spacing configs also do not work as desired.__
- __Might be duplicated; but no solution found...__

#### tslint console output

```ts
Could not find implementations for the following rules specified in the configuration:
    import-destructuring-spacing
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
```

#### Actual behavior
Missing spaces; but no error... import {Injectable} from '@angular/core';

#### Expected behavior
Correct formatting... import { Injectable } from '@angular/core';
Or lint error if spaces are missing.

**Comment by ajafff** [comment]
I guess you just upgraded `codelyzer` to v3.x?

https://github.com/mgechev/codelyzer/issues/262#issuecomment-284799722

**Comment by lunanigra** [comment]
Hello, that might be the right direction... Yes, we upgraded to codelyzer 3.0.0-beta.4; the following error/warning seems to confirm your guess...

```ts
codelyzer@3.0.0-beta.4 requires a peer of tslint@^4.0.0 but none was installed.
```

**Comment by ajafff** [comment]
There's not much we can do about this. The rule you are missing used to be provided by the 3rd party package `codelyzer`, which now dropped that rule. tslint currently has no equivalent.

Someone could contribute that functionality to the `whitespace` rule, though.

**Comment by adidahiya** [comment]
the whitespace rule's "check-module" option ought to do this, according to [its tests](https://github.com/palantir/tslint/blob/db83bfa043afcdae971db3967b6ed9279c2b3a55/test/rules/whitespace/all/test.ts.lint#L107). it might be broken, see https://github.com/palantir/tslint/issues/2401
